simdb/db.py

```diff
@@ -28,6 +28,8 @@
 PY2 = sys.version_info.major == 2
 
 try:
+    from pymor.core.pickle import dumps, load
```

Comment:
I would prefer to not let have simdb any special cases for pyMOR. One could think of some mechanism, however, to customize pickling.